src/field/captcha/third_party_captcha.jsx

```diff
@@ -31,9 +33,25 @@ const getCaptchaProvider = provider => {
       return window.friendlyChallenge;
     case ARKOSE_PROVIDER:
       return window.arkose;
+    case AUTH0_V2_CAPTCHA_PROVIDER:
+      return window.turnstile;
   }
 };
 
+export const getRenderParams = (self) => {
```

Comment:
Can we move this as part of the React component? We shouldn't be passing around `this` outside the component when we can have it as a class method for the component.